app/sprinkles/admin/templates/forms/user.html.twig

```diff
@@ -24,7 +24,8 @@
                         {% if 'group' in form.fields.disabled %}
                             <input type="text" class="form-control" name="theme" value="{{user.group.name}}" disabled>
                         {% else %}
-                            <select id="input-group" class="form-control js-select2" name="group_id">
+                            <select id="input-group" class="form-control js-select2-group" name="group_id">
```

Comment:
`js-select2-group` is not defined in this PR, making the select not select2 styled. 

Plus, when applying select2, the "no group" styling is weird. Should probably says "no group" or something.

<img width="299" alt="Capture d’écran, le 2019-07-10 à 21 26 15" src="https://user-images.githubusercontent.com/2566513/61015404-8167ef80-a359-11e9-96f7-bc17407ce53e.png">

Reply:
Fixed in b26b70512089f7f21cd3d09f609684edf95cf9c5

Reply:
The `No group`  will be the same formatting as the others...  Does Select2 support having a `<i>` inside the `option` element?

Reply:
Oops... It does not look like `<i>` is supported inside `option`